test/integration/connection/test-load-data-infile-disable.js

```diff
@@ -26,7 +26,8 @@ common.getTestConnection({localInfile: false}, function (err, connection) {
 
   connection.query(sql, [path, table, ',', newline], function (err) {
     assert.ok(err);
-    assert.equal(err.code, 'ER_NOT_ALLOWED_COMMAND');
+    // mariadb return 4166 error, mysql 1148
```

Comment:
The assert should assert the correct one for each, not just let both codes for both different databases.